.github/workflows/cxx.yml

```diff
@@ -60,38 +59,28 @@ jobs:
           - { os: macOS-10.15, target: msvc-2019 }
           - { os: macOS-10.15, target: msvc-2017 }
           - { os: macOS-10.15, target: msvc-2015 }
-          - { os: macOS-10.15, target: msvc-2013 }
           # We don't test g++ on Windows as DMD only mangles for MSVC
           - { os: windows-2019, target: g++-9 }
           - { os: windows-2019, target: g++-8 }
           - { os: windows-2019, target: g++-7 }
           - { os: windows-2019, target: g++-6 }
           - { os: windows-2019, target: g++-5 }
 
-          # TODO: Implement support for clang and MSVC on Windows
+          # TODO: Implement support for clang and MSVC2017 on Windows
           # Currently those are still being run by the auto-tester
-          - os: windows-2019
-
-        include:
-          # Clang boilerplate
-          - { target: clang-9.0.0, compiler: clang, cxx-version: 9.0.0 }
-          - { target: clang-8.0.0, compiler: clang, cxx-version: 8.0.0 }
-          - { target: clang-7.0.0, compiler: clang, cxx-version: 7.0.0 }
-          - { target: clang-6.0.0, compiler: clang, cxx-version: 6.0.0 }
-          - { target: clang-5.0.2, compiler: clang, cxx-version: 5.0.2 }
-          - { target: clang-4.0.0, compiler: clang, cxx-version: 4.0.0 }
-          - { target: clang-3.9.0, compiler: clang, cxx-version: 3.9.0 }
-          # g++ boilerplace
-          - { target: g++-9, compiler: g++, cxx-version: 9.3.0 }
-          - { target: g++-8, compiler: g++, cxx-version: 8.4.0 }
-          - { target: g++-7, compiler: g++, cxx-version: 7.5.0 }
-          - { target: g++-6, compiler: g++, cxx-version: 6.5.0 }
-          - { target: g++-5, compiler: g++, cxx-version: 5.5.0 }
-          # Platform boilerplate
-          - { os: ubuntu-16.04, arch: x86_64-linux-gnu-ubuntu-16.04 }
-          - { os: macOS-10.15,  arch: x86_64-apple-darwin }
-          # Clang 9.0.0 have a different arch for OSX
-          - { os: macOS-10.15, target: clang-9.0.0, arch: x86_64-darwin-apple }
```

Comment:
Uhoh this shouldn't have been removed... https://github.com/dlang/druntime/pull/3033